Sources/MapboxCoreNavigation/RouteController.swift

```diff
@@ -903,6 +937,32 @@ extension RouteController: ReroutingControllerDelegate {
     }
 }
 
+extension RouteController: AlternativeRoutesCenterDelegate {
+    func alternativeRoutesCenter(_ center: AlternativeRoutesCenter, didReportNewAlternatives updatedIndices: IndexSet, removedAlternatives: [AlternativeRoute]) {
+        let updatedAlternatives = updatedIndices.compactMap { alternativeRoutesCenter?.alternatives[$0] }
```

Comment:
Do we need `updatedIndices` if we immediately convert it to an array of alternatives? It seems that it's better to return the actual array in `AlternativeRoutesCenter`.

Reply:
My thinking was that user is able to get updated list through the `AlternativeRoutesCenter`, while indices will allow him to get the delta of what's changed if he needs to.

Reply:
But `AlternativeRoutesCenter` isn't public

Reply:
Right. It was public before, but I forgot to revisit this when updating the PR. So now I should either forward both updated array and new items indices to the end user, or just drop this indices idea at all. Do you think it is useless?

Reply:
Decided to not bother keeping the indices in the end.